extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -40,7 +40,9 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
       'click',
       this.view.setHandler(el => this.actionAddIntroHandler(el), this.view.errModule.handle(`add intro`))
     );
-    this.initSquire(this.isRichText());
+    if (this.isRichText()) {
+      this.initSquire(true);
+    }
```

Comment:
in this case Squire probably won't be re-initialized when user turns off rich text mode?

I think it'll be better to use `removeRichTextFormatting` here, which checks if `isRichText` is enabled before re-initializing squire:
```ts
this.isRichText() ? this.addRichTextFormatting() : this.removeRichTextFormatting();
```

Reply:
Hi @sosnovsky - I missed this scenario but I have corrected it now.